.env.dist

```diff
@@ -16,19 +16,21 @@ DEBUG='provider:*,registrar'
 REG_WS_API_URL=https://webservicesqa.it.ucla.edu
 REG_WS_CERT_NAME=/replace/with/pathto/file.cer
 REG_WS_PRIVATE_KEY=/replace/with/pathto/file.key
-REG_WS_CLIENT_ID=
-REG_WS_SECRET=
+SECRET_REG_WS_CLIENT_ID=
+SECRET_REG_WS_SECRET=
```

Comment:
@chern Strange to have SECRET_*_SECRET

Just rename it to SECRET_REG_WS_PASSWORD